packages/map-template/src/App.jsx

```diff
@@ -13,5 +14,5 @@ function App() {
     );
 }
 
-export default App;
+export default Sentry.withProfiler(App, { name: "MapsIndoorsMap" });
```

Comment:
This is really neat; you can see exactly when a component has been loaded using the "Spans" feature in Sentry, and filtering by the `ui.react.mount` option here: https://mapspeople-gx.sentry.io/performance/summary/spans/ui.react.mount:3112e291a8e98fa9/?project=4506851619831808&query=&statsPeriod=1h&transaction=%2F

![image](https://github.com/MapsPeople/web-ui/assets/136247/9ac0969d-b0ce-4d47-997f-28759610f81e)